src/core/server/elasticsearch/elasticsearch_service.test.ts

```diff
@@ -69,6 +70,27 @@ describe('#setup', () => {
     );
   });
 
+  it('returns data and admin client as a part of the contract', async () => {
+    const mockAdminClusterClientInstance = elasticsearchServiceMock.createClusterClient();
+    const mockDataClusterClientInstance = elasticsearchServiceMock.createClusterClient();
+    MockClusterClient.mockImplementationOnce(
+      () => mockAdminClusterClientInstance
+    ).mockImplementationOnce(() => mockDataClusterClientInstance);
+
+    const setupContract = await elasticsearchService.setup(deps);
```

Comment:
Do we have any reasons to keep this now?

https://github.com/elastic/kibana/blob/bde28957d997b9dd3f4beaac0b52b624d22d6f5c/src/core/server/elasticsearch/elasticsearch_service.ts#L61-L68

Reply:
As I understand, our code is ready for runtime config updates, but we don't have such requirements at the moment.